GoSSB/Sources/Makefile

```diff
@@ -57,11 +57,11 @@ GOEXTRAFLAGS=-trimpath
 GOBUILDTAGS = nommio x509omitbundledroots
 # ifdef DEBUG <  this var is not set when the Build Step script is called  by Xcode
 # ifeq ($(ENABLE_TESTABILITY),YES) < also works, not sure what the difference is
-ifeq ($(CONFIGURATION),Debug)
+#ifeq ($(CONFIGURATION),Debug)
     GOBUILDTAGS += testing
-else
+#else
     GOEXTRAFLAGS += -ldflags="-s -w"
-endif
+#endif
```

Comment:
These changes here in the Makefile fix https://github.com/planetary-social/planetary-ios/issues/719. Basically what was happening is that when we built GoSSB for a release it would exclude the testing functions. Since we commit the .a files to the repo now this would break the unit tests on CI. I commented out the macros so now every build includes the test functions and the ldflags. The ldflags strip DWARF debug symbols and the Go symbol table, which I think basically means you can't use gdb to debug the library. But I don't think anyone has been using that.

I know this isn't ideal but it solves our problems in the short term, and in the long term we will be switching to scuttlego and re-evaluating how we build and integrate Go in Xcode. CC @boreq is this change ok with you?